### PR TITLE
Fix naming strategy docs

### PR DESCRIPTION
**Pull request by krismas**

### CODE REVIEW

File: core/path-naming-strategy.md [hunk 1]
```diff
@@ -29,17 +29,17 @@ Let's assumes we need urls without separators (e.g. `api.tld/myresources`)
...
 
-Make sure the custom generator implements `ApiPlatform\Core\Routing\ResourcePathGeneratorInterface`:
+Make sure the custom generator implements [`ApiPlatform\Core\Naming\ResourcePathNamingStrategyInterface`](https://github.com/api-platform/core/blob/master/src/Naming/ResourcePathNamingStrategyInterface.php):
 
 ```php
-// src/AppBundle/Routing/NoSeparatorsResourcePathGenerator.php
+// src/AppBundle/Naming/NoSeparatorsResourcePathGenerator.php
 
-namespace AppBundle\Routing;
+namespace AppBundle\Naming;
```
Comment (teohhanhui) [inline core/path-naming-strategy.md]: Actually, by convention I'd place this under `AppBundle\ApiPlatform\Naming`. What do the others think? @api-platform/core-team

**Comment by krismas** [comment]
As far as I could see, in the documentation, most examples use this convention: `AppBundle\DataProvider`, `AppBundle\Action`...

File: core/path-naming-strategy.md (hunk 1 above)
Comment (soyuka) [inline core/path-naming-strategy.md]: I'm using a bundle on top of api-platform so `ApiBundle\Naming`. But, for dataproviders and such I'm simply doing `AppBundle\DataProvider`, so I'd keep `AppBundle\Naming` in the docs.

**Comment by teohhanhui** [comment]
Okay that should be a separate discussion.

Thanks @krismas!

**Pull request merged**